src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/VilkårService.kt

```diff
@@ -110,7 +109,7 @@ class VilkårService(
         val behandlingId = vilkår.behandlingId
 
         // TODO burde slettemarkeres hvis opprettet i tidligere behandling?
```

Comment:
Ref TODO her
Er tanken at vi også skal slettemarkere disse og ikke slette de helt når det gjelder et vilkår som ble opprettet i forrige behandling?

Reply:
Nei, siden fremtidig utgift ikke er et "ordentlig" vilkår men mer en kladd tenker jeg vi ikke trenger å slettemarkere og at vi kan slette de helt. Kan oppdatere TODOen, tenker det fortsatt kan gjelde for "vanlige" vilkår